Adding Scatterplot widget doc (#3737)
DataDog/documentation: content/graphing/dashboards/widgets.md

Let's say you are on a Timeseries and you are currently displaying the past hour, this button allows you to either display the `avg` / `max` / `min` / `avg` / `sum` / `last value` of ALL points that are rendered during that 1 hour range timeframe - depending on the aggregation chosen above.

## Scatterplot

*Supported on Screenboards and Timeboards*

The Scatterplot visualization allows you to graph a chosen scope over 2 different metrics with their respective aggregation:

> Select a metric and an aggregation for the X vs. the Y axis, then define the scope for each point of the Scatterplot e.g. `host`, `service`, `app`, `region`...

Reviewer: ```suggestion
Select a metric and an aggregation for the X and Y axis, then define the scope for each point of the Scatterplot, such as `host`, `service`, `app`, `region`, etc.
```
